site/layouts/partials/navbar-version-selector.html

```diff
@@ -0,0 +1,15 @@
+{{ $page_rel := .Page.RelPermalink | safeURL }}
+<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+	{{ .Site.Params.version_menu }}
+</a>
+<div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+	{{ range .Site.Params.versions }}
+		{{ $ver := .version }}
```

Comment:
Please replace all tabs with spaces.
P.S. VS Code can be easily configured to automate this: https://code.visualstudio.com/docs/editor/codebasics#_indentation